### PR TITLE
spring cleaning

### PR DESCRIPTION
**Pull request by vaibhavmule**
tests are successfully except for `tests/testing/test_unit_test.py` and `tests/test_upload_manager.py`

@josephmancuso can you have a look?

### CODE REVIEW

File: config/middleware.py [hunk 1]
```diff
@@ -1,51 +1,41 @@
...
+"""HTTP Middleware
+HTTP middleware is middleware that will be ran on every request. Middleware
+is only ran when a HTTP call is successful (a 200 response). This list
+should contain a simple aggregate of middleware classes.
 """
 
 HTTP_MIDDLEWARE = [
     LoadUserMiddleware,
     # todo
     # CsrfMiddleware,
```
Comment (vaibhavmule) [inline config/middleware.py]: @josephmancuso FYI. I have kept it as todo.

File: config/database.py [hunk 2]
```diff
@@ -47,3 +60,16 @@
...
+logger.setLevel(logging.DEBUG )
+
+formatter = logging.Formatter(
+    'It took %(elapsed_time)sms to execute the query %(query)s'
+)
+
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+
+logger.addHandler(handler)
```
Comment (josephmancuso) [inline config/database.py]: What is this addition?

File: config/middleware.py (hunk 1 above)
Comment (josephmancuso) [reply config/middleware.py]: to uncomment it?

File: config/storage.py [hunk 3]
```diff
@@ -1,84 +1,79 @@
...
     },
     's3': {
-        'client': os.getenv('S3_CLIENT', 'AxJz...'),
-        'secret': os.getenv('S3_SECRET', 'HkZj...'),
-        'bucket': os.getenv('S3_BUCKET', 's3bucket'),
-        'location': 'http://s3.amazon.com/bucket',
-    }
+        'client': env('S3_CLIENT', 'AxJz...'),
+        'secret': env('S3_SECRET', 'HkZj...'),
+        'bucket': env('S3_BUCKET', 's3bucket'),
```
Comment (josephmancuso) [inline config/storage.py]: The reason the tests are failing is because you are missing the `location` here

File: config/middleware.py (hunk 1 above)
Comment (vaibhavmule) [reply config/middleware.py]: I mean I kept it as it is. it commented out when I doing spring cleaning.

File: config/database.py (hunk 2 above)
Comment (vaibhavmule) [reply config/database.py]: I'm not sure but they are in masonite repo.
https://github.com/MasoniteFramework/masonite/blob/8f7a289e1e918b58c1115d669bea8fbe9b8a8a68/config/database.py#L55

Comment (josephmancuso) [reply config/database.py]: 🤔 i'm not sure why i added these .... lol

File: config/storage.py (hunk 3 above)
Comment (vaibhavmule) [reply config/storage.py]: thanks for pointing out

File: config/database.py (hunk 2 above)
Comment (vaibhavmule) [reply config/database.py]: is it related to logger driver?

Comment (josephmancuso) [reply config/database.py]: no it's suppose to show the queries in the terminal (so you can optimize) but i thought thats what the `DB_LOG` was doing .. lol

Comment (josephmancuso) [reply config/database.py]: not sure why i added this extra code

Comment (vaibhavmule) [reply config/database.py]: we can keep it for now and remove it later.

**Comment by vaibhavmule** [comment]
@josephmancuso Now it is failing at session, did i miss something?
https://travis-ci.org/MasoniteFramework/core/jobs/477381795

File: config/session.py [hunk 4]
```diff
@@ -1,18 +1,13 @@
...
+"""Session Driver
+Sessions are able to be linked to an individual user and carry data from
+request to request. The memory driver will store all the session data
+inside memory which will delete when the server stops running.
+
+Supported: 'memory', 'cookie'
 """
 
-DRIVER = os.getenv('SESSION_DRIVER', 'memory')
+DRIVER = env('SESSION_DRIVER', 'cookie')
```
Comment (josephmancuso) [inline config/session.py]: possibly this change from cookie to memory?

Comment (josephmancuso) [reply config/session.py]: this shouldn't break it tho but i wonder if this is breaking it

Comment (josephmancuso) [reply config/session.py]: does it work locally?

Comment (vaibhavmule) [reply config/session.py]: No it does not, memory works.

Comment (josephmancuso) [reply config/session.py]: hmm i'm gonna need to bring that in later and make sure nothing is wrong with that

File: config/database.py (hunk 2 above)
Comment (vaibhavmule) [reply config/database.py]: ?

**Pull request merged**